trin-core/src/bin/generate-data.rs

```diff
@@ -44,16 +46,17 @@ pub fn main() -> Result<(), Box<dyn std::error::Error>> {
         node_id,
         distance_function: DistanceFunction::Xor,
         db: Arc::new(db),
-        meta_db: Arc::new(meta_db),
+        sql_handler_tx,
     };
-    let mut storage = PortalStorage::new(storage_config, |key| sha256(key))?;
+    let mut storage = PortalStorage::new(storage_config).await?;
 
     for _ in 0..num_of_entries {
         let value = generate_random_value(size_of_values);
         let key = generate_random_value(SIZE_OF_KEYS);
 
-        println!("{} -> {}", &key, &value);
-        storage.store(&key, &value)?;
+        let content_key = ContentKey::StateContentKey(StateContentKey::from_bytes(&key).unwrap());
```

Comment:
Hahah yeah good catch, random bytes should fail pretty much every time. I sort of glossed over this script since it's mostly obsolete given #228 but I've added a proper error handling for now just in case anybody uses it in the meantime. Though it's also pretty useless if it'll only write random bytes that happen to start with the few possible selectors... but I'm quite confident nobody's actually making use of this script right now.... but if you disagree or know anybody who is lmk - and I'll change it so that it's a bit more useful